app/_redirects

```diff
@@ -3,6 +3,14 @@
 # More general/sweeping redirects should be at the bottom.
 
 
+
+
+# Konnect rename
+
+/konnect/runtime-manager/* /konnect/gateway-manager/:splat
+/konnect/runtime-manager/runtime-instances/* /konnect/gateway-manager/data-plane-nodes/:splat
+/konnect/runtime-manager/composite-runtime-groups/*     /konnect/gateway-manager/control-plane-groups/:splat
```

Comment:
Same issue with this one as the one above. `gateway-manager` part redirects, but it doesn't change to `control-plane-groups` as well, so there's a 404.